recipes/catch2/2.x.x/conanfile.py

```diff
@@ -50,4 +50,6 @@ def package_id(self):
         self.info.header_only()
 
     def package_info(self):
-        self.cpp_info.builddirs = [os.path.join("lib", "cmake", "Catch2")]
```

Comment:
please, revert it. We can fix the hook

Reply:
Okay. I'm not sure that having a resulted info be dependent on the platform it was built on is a good way though.